identity-service/src/txRelay.js

```diff
@@ -28,16 +27,24 @@ async function delay (ms) {
 }
 
 async function getGasPrice (req, web3) {
+  // If req object logger is present, use it to also print out request ID. Otherwise, use the global logger
```

Comment:
instead of passing req through here and having this check both in this function and createAndSendTransaction, you could just pass in logger as a function param

Reply:
ah that makes a lot more sense. updating!